breadcord/helpers.py

```diff
@@ -207,18 +209,19 @@ def decorator(cls: type[_Transformer]) -> _Transformer:
     return decorator
 
 
-class HTTPModuleCog(breadcord.module.ModuleCog):
+class HTTPModuleCog(ModuleCog):
```

Comment:
Changed becasue I was getting some weird recursive import error, not sure why, didn't happen before, but I would prefer for it not to happen in production